### ISSUE TITLE
37 basic plots

### ISSUE THREAD

**Issue by oelna** [oelna/mainstream-screenplay-generator#1]
http://www.openculture.com/2020/08/37-possible-stories.html

**Comment by Incredibul** [comment]
Ja, als ich vorm Studium noch nerdig RPG Spielleiter war gab's da öfter
einen Verweis drauf 😅
https://www.rpglibrary.org/articles/storytelling/36plots.php

Arno Richter <notifications@github.com> schrieb am Mo., 31. Aug. 2020,
22:09:
